test/fixtures/waitUntil.conf.js

```diff
@@ -1,5 +1,8 @@
 var path = require('path')
+
 var chai = require('chai')
+var chaiString = require('chai-string')
+var chaiAsPromised = require('chai-as-promised')
```

Comment:
do we really need this duplication in all fixtures? We already have it in mocha setup scripts